libs-scala/postgresql-testing/src/main/scala/com/digitalasset/testing/postgresql/PortLock.scala

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.testing.postgresql
+
+import java.io.RandomAccessFile
+import java.nio.channels.{
+  ClosedChannelException,
+  FileChannel,
+  FileLock,
+  OverlappingFileLockException
+}
+import java.nio.file.{Files, Path, Paths}
+
+import com.digitalasset.ports.Port
+
+private[postgresql] object PortLock {
+
+  // We can't use `sys.props("java.io.tmpdir")` because Bazel changes this for each test run.
+  // For this to be useful, it needs to be shared across concurrent runs.
+  private val portLockDirectory: Path = {
+    val tempDirectory =
+      if (sys.props("os.name").startsWith("Windows")) {
```

Comment:
What about `sys.props("java.io.tmpdir")`?

Reply:
That's loaded from `$TMPDIR`, which Bazel overrides.